src/orderbook.ts

```diff
@@ -1,6 +1,6 @@
 import {Order} from ".";
 
-export class Price {
+export class Fraction {
```

Comment:
Also, I just remembered, TS has this cool feature:
```ts
export class Fraction {
    constructor(
        public readonly numerator: number,
        public readonly denominator: number,
    ) {
        if (denominator === 0) {
            throw new Error("error");
        }
    }
}
```

This will make both the properties read-only and automagically set their value from the constructor arguments.